README.md

```diff
@@ -22,6 +22,9 @@ GH_CLIENT_ID=xyz123
 GH_CLIENT_SECRET=asdf123
 # The URL to redirect the user to once the authentication was successful
 REDIRECT_URL=https://google.com
+
```

Comment:
Will do. My logic in the separate line was to differentiate it from the required params. Line removal works for me!